django/cantusdb_project/main_app/admin.py

```diff
@@ -22,7 +23,15 @@ class CenturyAdmin(BaseModelAdmin):
 
 
 class ChantAdmin(BaseModelAdmin):
-    exclude = EXCLUDE + ("col1", "col2", "col3", "next_chant", "s_sequence", "is_last_chant_in_feast")
+    exclude = EXCLUDE + (
+        "col1",
+        "col2",
```

Comment:
I'm confused by why black keeps formatting unrelated files? 

The PR looks good but that behavior doesn't make sense to me...